handlers/indexs3.go

```diff
@@ -0,0 +1,51 @@
+package handlers
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// GetIndexdRecordRev gets record rev
+func GetIndexdRecordRev(uuid, indexURL string) (string, error) {
+	req, err := http.NewRequest("GET", indexURL+"/"+uuid, nil)
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		panic(err)
+	}
+	defer resp.Body.Close()
+
+	fmt.Println("response Status:", resp.Status)
+	if resp.StatusCode != 200 {
+		return "", errors.New("Can not get record rev")
```

Comment:
could include UUID and URL information in the error

Reply:
Fixed in ba795498eb1e0537bae5bb2abb54fe15b7f7e6c5